velox/functions/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ add_library(velox_function_registry FunctionRegistry.cpp)
 add_library(velox_coverage_util CoverageUtil.cpp)
 
 target_link_libraries(velox_function_registry velox_expression velox_type
-                      velox_core velox_exception)
+                      velox_core velox_exception velox_exec)
```

Comment:
Why do you have a dependency from velox_function_registry to velox_exec ? There shouldn't be any dependencies from the velox functions code to velox_exec.

Reply:
Hi @aditi-pandit, this change was added to fix the below linker error:
```
ld: Undefined symbols:
  facebook::velox::exec::windowFunctions(), referenced from:
      facebook::velox::getWindowNames(bool) in libvelox_function_registry.a[2](FunctionRegistry.cpp.o)
  facebook::velox::exec::aggregateFunctions(), referenced from:
      facebook::velox::getScalarNames(bool) in libvelox_function_registry.a[2](FunctionRegistry.cpp.o)
      facebook::velox::getAggregateNames(bool) in libvelox_function_registry.a[2](FunctionRegistry.cpp.o)
      facebook::velox::getWindowNames(bool) in libvelox_function_registry.a[2](FunctionRegistry.cpp.o)
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```